src/index.html

```diff
@@ -13,6 +13,38 @@
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-mon calendar--month-length-31">
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), BEM block modifiers should be separated by double underscore `__` instead of `--`. Please correct the class names for `calendar--start-day-mon` and `calendar--month-length-31` to use the BEM naming convention correctly.